### PR TITLE
Fix numpy warning (round 2)

### PR DESCRIPTION
**Pull request by arnaudbore**
- np.asarray needs dtype

### CODE REVIEW

**Comment by scil-admin** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-474/1/

**Comment by scil-admin** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-474/2/

File: scilpy/tracking/tools.py [hunk 1]
```diff
@@ -80,7 +81,7 @@ def get_subset_streamlines(sft, max_streamlines, rng_seed=None):
     ind = np.arange(len(sft.streamlines))
     rng.shuffle(ind)
 
-    subset_streamlines = list(np.asarray(sft.streamlines)[
+    subset_streamlines = list(np.asarray(sft.streamlines, dtype=np.float)[
```
Comment (boiteaclou) [inline scilpy/tracking/tools.py:84]:
```suggestion
    subset_streamlines = list(np.asarray(sft.streamlines, dtype=object)[
```

**Comment by scil-admin** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-474/3/

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Review by boiteaclou — APPROVED** [review]

**Pull request merged**